### ISSUE TITLE
Bugs: generated sig for mailer method doesn't include default value

### ISSUE THREAD

**Issue by manhhung741** [chanzuckerberg/sorbet-rails#192]
**Describe the bug:**
```
  def notify(parent_id:, login_token: nil, should_update_sent_flag: true)
```
Generate 
```
  sig { params(parent_id: T.untyped, login_token: T.untyped, should_update_sent_flag: T.untyped).returns(ActionMailer::MessageDelivery) }
  def self.notify(parent_id:, login_token:, should_update_sent_flag:); end
```

**Steps to reproduce:**

**Expected behavior:**

**Versions:**

- Ruby:
- Rails:
- Sorbet:
- Sorbet-Rails:

**Comment by jaredbeck** [comment]
Harry's example demonstrates optional keyword arguments. There is a similar issue with regular optional arguments.

```
def redacted(subject, to, message, from = REDACTED, category = :REDACTED)
```

generates:

```
  sig do
    params(
      subject: T.untyped,
      to: T.untyped,
      message: T.untyped,
      from: T.untyped,
      category: T.untyped
    ).returns(ActionMailer::MessageDelivery)
  end
  def self.redacted(subject, to, message, from, category); end
```

**Comment by manhhung741** [comment]
Yeah :( I don’t know a way to retrieve the default values of parameters.
For now, I’m fixing them by hand. Maybe we should have a way to skip those
until there is an actual solution

**Comment by jaredbeck** [comment]
> For now, I’m fixing them by hand.

Same here (editing eg. `sorbet/rails-rbi/mailers/my_mailer.rbi`)

**Comment by manhhung741** [comment]
FYI @jaredbeck, the issue is fixed in sorbet 0.6.1! (just released today)

**Comment by jaredbeck** [comment]
Thanks Harry! :heart:
